src/core/src/op/mvn.cpp

```diff
@@ -135,3 +136,84 @@ bool op::v6::MVN::visit_attributes(AttributeVisitor& visitor) {
+    } else if (inputs[1]->get_element_type() == element::i32) {
+        reduction_axes = runtime::reference::mvn_6_reduction_axes<int32_t>(inputs[1], rank);
+    } else {
+        throw ngraph_error("Unexpected indices type");
+    }
+    runtime::reference::mvn_6<T>(inputs[0]->get_data_ptr<ET>(),
+                                 outputs[0]->get_data_ptr<ET>(),
+                                 inputs[0]->get_shape(),
+                                 reduction_axes,
+                                 normalize_variance,
+                                 eps,
+                                 eps_mode);
+    return true;
+}
+
+bool evaluate_mvn(const HostTensorVector& outputs,
+                  const HostTensorVector& inputs,
+                  bool normalize_variance,
+                  float eps,
+                  ov::op::MVNEpsMode eps_mode) {
+    bool rc = true;
+    switch (inputs[0]->get_element_type()) {
+        NGRAPH_TYPE_CASE(evaluate_mvn, i8, outputs, inputs, normalize_variance, eps, eps_mode);
+        NGRAPH_TYPE_CASE(evaluate_mvn, i16, outputs, inputs, normalize_variance, eps, eps_mode);
+        NGRAPH_TYPE_CASE(evaluate_mvn, i32, outputs, inputs, normalize_variance, eps, eps_mode);
+        NGRAPH_TYPE_CASE(evaluate_mvn, i64, outputs, inputs, normalize_variance, eps, eps_mode);
+        NGRAPH_TYPE_CASE(evaluate_mvn, u8, outputs, inputs, normalize_variance, eps, eps_mode);
+        NGRAPH_TYPE_CASE(evaluate_mvn, u16, outputs, inputs, normalize_variance, eps, eps_mode);
+        NGRAPH_TYPE_CASE(evaluate_mvn, u32, outputs, inputs, normalize_variance, eps, eps_mode);
+        NGRAPH_TYPE_CASE(evaluate_mvn, u64, outputs, inputs, normalize_variance, eps, eps_mode);
+        NGRAPH_TYPE_CASE(evaluate_mvn, bf16, outputs, inputs, normalize_variance, eps, eps_mode);
+        NGRAPH_TYPE_CASE(evaluate_mvn, f16, outputs, inputs, normalize_variance, eps, eps_mode);
+        NGRAPH_TYPE_CASE(evaluate_mvn, f32, outputs, inputs, normalize_variance, eps, eps_mode);
+    default:
+        rc = false;
+        break;
+    }
+    return rc;
+}
+}  // namespace
+}  // namespace mvn
+
+bool op::v6::MVN::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const {
+    OV_OP_SCOPE(v6_MVN_evaluate);
+    return mvn::evaluate_mvn(outputs, inputs, get_normalize_variance(), get_eps(), get_eps_mode());
+}
+
+bool op::v6::MVN::has_evaluate() const {
+    OV_OP_SCOPE(v1_MVN_has_evaluate);
+    switch (get_input_element_type(0)) {
+    case ngraph::element::i8:
+    case ngraph::element::i16:
+    case ngraph::element::i32:
+    case ngraph::element::i64:
+    case ngraph::element::u8:
+    case ngraph::element::u16:
+    case ngraph::element::u32:
+    case ngraph::element::u64:
+    case ngraph::element::bf16:
+    case ngraph::element::f16:
+    case ngraph::element::f32:
+        return true;
+    default:
+        break;
+    }
+    return false;
+}
```

Comment:
do we need tests for newly added methods?

Reply:
I think we need to change template_plugin func tests to use evaluate instead of evaluate map

Reply:
Removed MVN6 evaluate() from evaluate map. So currently template_plugin func tests should run evaluate from ov::core.

Reply:
Looks like we need evaluate() in evaluates_map as well, as there are some tests for non-float types which stop to work if I remove it.

Reply:
We need to have some tests for new functions

Reply:
I have added constant folding tests.